### PR TITLE
product data loaded

### PR DESCRIPTION
**Pull request by pritikc**
Team ,

please review this pr , for product changes

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!-- <h1 class="container text-center">Practise Projects</h1> -->
 
-<hr>
+<!-- <hr> -->
```
Comment (catchsrini2004) [inline src/app/app.component.html:3]: remove unnecesary commented lines

File: src/app/data-share.service.ts [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
   providedIn: 'root'
 })
 export class DataShareService {
 
-  constructor() { }
+  constructor(private httpCall:HttpClient) { }
+
+
+callGetData(){
+  let dataResult = this.httpCall.get('https://dummyjson.com/products');
```
Comment (catchsrini2004) [inline src/app/data-share.service.ts:13]: add the URL in environment file or in constant variable

File: src/app/home/home.component.html [hunk 3]
```diff
@@ -12,7 +12,35 @@
     <div class="col items">
         <button>CARTS</button>
     </div>
-</div>
+</div> -->
+
+<!-- --- -->
```
Comment (catchsrini2004) [inline src/app/home/home.component.html:17]: remove this emty line

File: src/app/products/products.component.ts [hunk 4]
```diff
@@ -7,4 +20,30 @@ import { Component } from '@angular/core';
...
+  errorFlag=false;
+  constructor(private ds:DataShareService){
+
+  
+
+  // loadProductData(){
+    
+  this.ds.callGetData().subscribe( 
+    
+    (productResult : any) => {debugger;
```
Comment (catchsrini2004) [inline src/app/products/products.component.ts]: Do not add debugger statement in code and commit

Comment (pritikc) [reply src/app/products/products.component.ts]: fixed this and commited changes

File: src/app/home/home.component.html (hunk 3 above)
Comment (pritikc) [reply src/app/home/home.component.html:17]: fixed

File: src/app/data-share.service.ts (hunk 2 above)
Comment (pritikc) [reply src/app/data-share.service.ts:13]: fixed

File: src/app/app.component.html (hunk 1 above)
Comment (pritikc) [reply src/app/app.component.html:3]: fixed

**Review by catchsrini2004 — APPROVED** [review]
approved

**Pull request merged**